### ISSUE TITLE
Bug: Deeplink not recognized from cold start on iOS

### ISSUE THREAD

**Issue by malua** [BranchMetrics/cordova-ionic-phonegap-branch-deep-linking-attribution#732]
### Describe the bug

Deeplinks are not correctly parsed when the is killed on iOS. They always are recognized as non-Branch links. When the app is background it's working correctly.  Sometimes it's works the first time when the app was killed, but it won't work anymore subsqequently.

### Steps to reproduce

- Open a deeplink when the app is closed 

```
this.branch
      .initSession()
      .then(data => {
        console.log('initSession', data);
```

` {"+clicked_branch_link: "false", "+is_first_session": "false"}`

- Open a deeplink when the is in background
Branch correctly identifies the link as a Branch link and we get the data.

### Expected behavior

I expect the Deeplink data to be parsed when the is killed before.

### SDK Version

Cordova 6.2.0 (from underlying iOS SDK Version 3.4.0 onwards)

### Make and Model

iPhone 14

### OS

iOS 17

### Additional Information/Context

I traced the issue to the ios SDK Version 3.4.0 https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/releases/tag/3.4.0

This is the first version with which it does not work anymore. If I downgrade to 3.3.0 in the pods it still works.

**Comment by malua** [comment]
Here is a video where the redirect works everytime with iOS SDK 3.3.0

https://github.com/user-attachments/assets/6a87f216-c22c-4052-b59a-18cadbec6fdc


Here it is with 3.4.0. It works only sometimes but most of the time the branch init function is called with 
 {"+clicked_branch_link: "false", "+is_first_session": "false"}


https://github.com/user-attachments/assets/d974a3e7-73c1-430c-8586-4459eb2167a3

**Comment by malua** [comment]
The bug is fixed in iOS SDK 3.6.2

https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/releases/tag/3.6.2

It would be great, if we could get the latest SDK updates also for Cordova. Thanks!

**Comment by malua** [comment]
Thank you!
